holidays/countries/syrian_arab_republic.py

```diff
@@ -0,0 +1,214 @@
+        if self._year >= 1963:
+            self._add_holiday_mar_8(tr("عيد الثورة"))
+
+        # Mother's Day.
+        if self._year >= 1950:
+            self._add_holiday_mar_21(tr("عيد الأم"))
+
+        # Independence Day.
+        if self._year >= 1946:
+            self._add_holiday_apr_17(tr("عيد الجلاء"))
+
+        # Labor Day.
+        if self._year >= 1948:
+            self._add_labor_day(tr("عيد العمال"))
+
+        # Martyrs' Day.
+        if self._year >= 1947:
+            self._add_holiday_may_6(tr("عيد الشهداء"))
+
+        # Anniversary of the October Liberation War.
+        if self._year >= 1974:
+            self._add_holiday_oct_6(tr("عيد حرب تشرين التحريرية"))
+
+        # Christmas Day.
+        self._add_christmas_day()
+        if self._year >= 2024:
+            self._add_christmas_day(tr("عيد الميلاد المجيد"), GREGORIAN_CALENDAR)
+
+        # Boxing Day.
+        if self._year >= 2024:
+            self._add_christmas_day_two(tr("يوم الصناديق"))
+
+        # Easter according to the Eastern calendar.
+        self._add_easter_sunday(tr("عيد الفصح حسب التقويم الشرقي"))
+
+        # Easter according to the Western calendar.
+        self._add_easter_sunday(tr("عيد الفصح حسب التقويم الغربي"), GREGORIAN_CALENDAR)
+
+        # Islamic Holidays.
+        name = tr("عيد الفطر")
+        self._add_eid_al_fitr_day(name)
+        self._add_eid_al_fitr_day_two(name)
+        self._add_eid_al_fitr_day_three(name)
+
+        self._add_arafah_day(tr("وقفة عرفة"))
+        aldha_name = tr("عيد الأضحى")
+        self._add_eid_al_adha_day(aldha_name)
+        self._add_eid_al_adha_day_two(aldha_name)
+        self._add_eid_al_adha_day_three(aldha_name)
+
+        self._add_islamic_new_year_day(tr("رأس السنة الهجرية"))
+        self._add_mawlid_day(tr("عيد المولد النبوي"))
+
+        # Isra' and Mi'raj.
+        self._add_isra_and_miraj_day(tr("الإسراء والمعراج"))
+
+
+class SY(SyrianArabRepublic):
+    pass
+
+
+class SYR(SyrianArabRepublic):
+    pass
+
+
+class SyrianArabRepublicIslamicHolidays(_CustomIslamicHolidays):
+    EID_AL_FITR_DATES = {
+        2017: (JUN, 25),
+        2018: (AUG, 22),
+        2019: (AUG, 11),
+        2020: (MAY, 24),
+        2021: (MAY, 13),
+        2022: (MAY, 2),
+        2023: (APR, 21),
+        2024: (APR, 10),
+        2025: (MAR, 31),
+    }
+
+    EID_AL_ADHA_DATES = {
+        2017: (SEP, 1),
+        2018: (AUG, 22),
+        2019: (AUG, 11),
+        2020: (JUL, 31),
+        2021: (JUL, 20),
+        2022: (JUL, 9),
+        2023: (JUN, 28),
+        2024: (JUN, 16),
+        2025: (JUN, 6),
+    }
+
+    HIJRI_NEW_YEAR_DATES = {
+        2017: (JUN, 25),
+        2018: (JUN, 15),
+        2019: (JUN, 4),
+        2020: (AUG, 20),
+        2021: (AUG, 9),
+        2022: (JUL, 30),
+        2023: (JUL, 19),
+        2024: (JUL, 7),
+        2025: (JUN, 27),
+    }
+
+    MAWLID_DATES = {
+        2017: (NOV, 30),
+        2018: (NOV, 18),
+        2019: (NOV, 9),
```

Comment:
Again, make sure that these Islamic holidays are accurate to Syria's own observance